### DISCUSSION TITLE
Is it possible to use multiple themes with prefix classes? (ex: coffee:bg-red-500)

### DISCUSSION THREAD

**Discussion by ahmetbarut** [tailwindlabs/tailwindcss#10868] [General]
Hi, I'm new to tailwind css and I like it a lot. When I started using the dark mode, such a question came to my mind. Is it possible to use themes with multiple prefixes?

```html
<div class="grid coffee:grid-cols-3 default:grid-cols-1 default:gap-2 coffee:gap-4 p-1">
    <div class="p-4 bg-blue-300"></div>
    <div class="p-4 bg-green-300"></div>
    <div class="p-4 bg-orange-300"></div>
</div>
```
### Like This
<img width="789" alt="Ekran Resmi 2023-03-26 18 09 44" src="https://user-images.githubusercontent.com/36641723/227785242-555089c0-bda5-4cfe-9a99-7fcf3104bf59.png">

**Comment by wongjn** [comment]
Yep! [Use a Tailwind Plugin to add your own variants](https://tailwindcss.com/docs/plugins#adding-variants).
